buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java

```diff
@@ -108,8 +109,12 @@ public String call() throws Exception {
             // Here we manually add any project dependencies in the "shadow" configuration to our generated POM
             publication.getPom().withXml(xml -> {
                 Node root = xml.asNode();
-                root.appendNode("name", project.getName());
-                root.appendNode("description", project.getDescription());
+                if (((NodeList) root.get("name")).isEmpty()) {
```

Comment:
May conflict with `com.netflix.nebula.maven-base-publish` which adds `name` and `description` properties